### ISSUE TITLE
Dialog component - accessibility violation.

### ISSUE THREAD

**Issue by DianeGit22** [Sage/carbon#4771]
### Current behaviour

Dialog component "Test - default" story reports 1 accessibility violation "Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds" as shown below. Note this issue occurs on Firefox but not on Chrome or Edge. 

![image](https://user-images.githubusercontent.com/47564230/151938820-02cbde7b-0a47-443f-b18b-ee4bf60c369b.png)

### Expected behaviour

Contrast between foreground and background colors should meet WCAG 2 AA contrast ratio threshold.

### CodeSandbox or Storybook URL

https://carbon.sbc-ui.com/?path=/story/dialog-test--default-story

### Carbon Version

Latest

### What browsers are you seeing the problem on?

Firefox

### What Operating System are you seeing the problem on?

Windows

### Confidentiality

- [X] I confirm there is no confidential or commercially sensitive information included.

**Comment by samtjo** [comment]
FE-4897

**Comment by divyajindel** [comment]
This issue is no longer with Dialog, not seeing any accessibility violation for now.
